### ISSUE TITLE
Batch delete for comments

### ISSUE THREAD

**Issue by vlhtg** [eddyizm/insta_delete#12]
I was looking for such a script to delete activity on Instagram, but I see that your script does only support deleting posts and likes.

I suggest that a good improvement would be doing the same for comments too. I will try to fork your code and maybe do it myself, but I have zero Selenium experience.

**Comment by eddyizm** [comment]
To be clear, you want to find/track your comments and then go and delete them? 
I can look at being able to delete your comment however finding and tracking where you are commenting is the bigger challenge.
I believe there is a activity tab that we should be able to scrape.

EDIT:  realized Ig now has the comments all set at a URL so rather easy to find them. Let me know if this is what you were looking for, I don't use IG much manually so it looks like it is feasible.

**Comment by yazanzaid00** [comment]
I'm encountering an issue when trying to select comments on my activity page. Visually, it appears that I've clicked the button, but the number of selections remains at one. 

The webpage I'm attempting to bulk delete Instagram comments from is: [https://www.instagram.com/your_activity/interactions/comments](https://www.instagram.com/your_activity/interactions/comments)

Here is the JavaScript code I am using:

```js
const buttons = document.querySelectorAll('div[data-bloks-name="bk.components.Flexbox"][aria-label="Toggle checkbox"]');
buttons.forEach(button => {
  button.click();
});
```

To select all the buttons really and not only visually. it selects them visually but they does not get registered by the website. instagram shows 1 selected instead of big number selected.

**Comment by bigplayer-ai** [comment]
I've run into an issue while attempting to select comments on my activity page. Visually, it seems that I've successfully clicked the button, but the count of selections remains at one.

The webpage from which I'm trying to bulk delete Instagram comments can be found here: [https://www.instagram.com/your_activity/interactions/comments](https://www.instagram.com/your_activity/interactions/comments)

Here is the JavaScript code I'm currently using:

```javascript
const buttons = document.querySelectorAll('div[data-bloks-name="bk.components.Flexbox"][aria-label="Toggle checkbox"]');
buttons.forEach(button => {
  button.click();
});
```

I'm looking to select all the buttons effectively, not just visually. Currently, the buttons are visually selected but aren't registered by the website. Instead of displaying a large number selected, Instagram shows only one selected.

**Comment by Ikhari** [comment]
Yes, delete all comments on this URL https://www.instagram.com/your_activity/interactions/comments

**Comment by eddyizm** [comment]
Might try one at a time and loop it but I don't really have a need for this feature so I may let it linger till I get some spare time.
